resources/playlist/playlistController.js

```diff
@@ -21,16 +21,15 @@ const getAllMusic = async (req, res) => {
       });
     }
   } catch (error) {
-    return res.status(500).json({
-      status: 500,
-      error,
-    });
+    res
+      .status(500)
+      .json({ message: `Error getting all music: ${error.message}` });
   }
 };
 
 const addMusic = async (req, res) => {
   try {
-    const { artist, title, video_url, favourite } = req.body;
+    const { artist, title, video_url, thumbnail_url, favourite } = req.body;
     const userId = req.userId;
     const { token } = req.headers;
     if (token) {
```

Comment:
Good job so far Joba, just a minor suggestion. I'm not sure why we have to check for the token in the controller again when we are passing validateToken as a middleware in the routes. I guess that's one of the primary aims of using the validateToken middleware. The token check should be implemented there